src/auth/auth.ts

```diff
@@ -11,9 +11,12 @@ import {
 import {
   BBAuthTokenResponse
 } from './auth-token-response';
+import { BBAuthGetUriArgs } from './auth-get-uri-args';
```

Comment:
Could you format the import the way the other imports are formatted with a blank line in between the import statements?.

```
import {
  BBAuthTokenResponse
} from './auth-token-response';

import {
  BBAuthGetUrlArgs
} from './auth-get-url-args';
```